src/methods/sep10Auth/start.ts

```diff
@@ -0,0 +1,44 @@
+import { Keypair, Transaction } from "stellar-sdk";
+import { log } from "helpers/log";
+
+export const start = async ({
+  authEndpoint,
+  secretKey,
+}: {
+  authEndpoint: string;
+  secretKey: string;
+}) => {
+  const publicKey = Keypair.fromSecret(secretKey).publicKey();
+  const params = { account: publicKey };
```

Comment:
SEP-10 supports a `home_domain` parameter that should be used when the SEP-10 server issues tokens to be used to authenticate with services on different domains. If its not too much work we should add a `forHomeDomain` parameter to the `start()` function and use it in the request.

Reply:
Added in https://github.com/stellar/stellar-demo-wallet/pull/125